pycape/function_ref.py

```diff
@@ -85,15 +69,45 @@ def checksum(self):
     def token(self):
         return self._token
 
-    @property
-    def auth_type(self):
-        return self._auth_type
+    @classmethod
+    def from_json(cls, token_path: Union[str, os.PathLike]):
```

Comment:
`to_json` would be good to have as well I think, this can be a TODO

Reply:
good idea!

Reply:
https://capeprivacy.atlassian.net/browse/CAPE-1048